runtime.md

```diff
@@ -13,6 +13,8 @@ The state of a container MUST include, at least, the following properties:
 * **`id`**: (string) is the container's ID.
 This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
+The ID is provided in the state because hooks will be executed with the state as the payload.
```

Comment:
This line was just [recovered from the graveyard][1].  It sounds like you want it to stay dead, so I'll drop it when I rebase.

[1]: https://github.com/opencontainers/runtime-spec/commit/be594153b522f52bebd0500cef6fe0f1d77f6a59#diff-b84a8d65d8ed53f4794cd2db7e8ea731L16